controllers/controllerArticles.js

```diff
@@ -8,4 +8,11 @@ function getArticleDataById(req, res, next) {
   .catch(next)
 }
 
-module.exports = { getArticleDataById };
+function getAllArticleData(req, res, next){
+    retrieveAllArticleData().then((result) => {
+        res.status(200).send(result)
```

Comment:
This should be going back as an object like this: `{article : result}` to make an api that is easy for developers (including yourself!) to understand and use